ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/product/PropertiesSection.java

```diff
@@ -130,8 +131,8 @@ private class PropertyDialog extends StatusDialog {
 
 		private final String[] COMBO_OSLABELS = new String[] { PDEUIMessages.PropertiesSection_All, Platform.OS_LINUX,
 				Platform.OS_MACOSX, Platform.OS_WIN32 };
-		@SuppressWarnings("deprecation")
-		private final String[] COMBO_ARCHLABELS = new String[] { PDEUIMessages.PropertiesSection_All, Platform.ARCH_X86,
+		private final String[] COMBO_ARCHLABELS = new String[] { PDEUIMessages.PropertiesSection_All,
+				IArgumentsInfo.ARCH_X86,
```

Comment:
What makes me wonder if we should add the other supported archs now here? so what about arm, longaarch and alike?